### PR TITLE
Fix for neovim 0.10

### PR DESCRIPTION
**Pull request by Shougo**
`health#report_XXX` is deprecated in neovim 0.10.

### CODE REVIEW

File: autoload/health/denops.vim [hunk 1]
```diff
@@ -21,105 +21,137 @@ function! s:get_deno_version(deno) abort
...
+    call health#report_warn(a:report)
+  endif
+endfunction
+
+function! s:report_error(report) abort
+  if has('nvim-0.10')
+    call v:lua.vim.health.error(a:report)
+  else
+    call health#report_error(a:report)
+  endif
```
Comment (lambdalisue) [inline autoload/health/denops.vim:154]: Please rewrite it like

```vim
if has('nvim-0.10')
  function! s:report_ok(report) abort
    call v:lua.vim.health.ok(a:report)
  endfunction

  " ...etc
else
  function s:report_ok(report) abort
    call healtp#report_ok(a:report)
  endfunction

  " ...etc
endif
```

**Review by lambdalisue — CHANGES_REQUESTED** [review]

File: autoload/health/denops.vim (hunk 1 above)
Comment (Shougo) [reply autoload/health/denops.vim]: Fixed.

**Review by lambdalisue — APPROVED** [review]
LGTM

**Comment by Shougo** [comment]
Thanks.

**Pull request merged**